### PR TITLE
[Merged by Bors] - Lighthouse crate v0.2.0 bump

### PR DESCRIPTION
**Pull request by AgeManning**
## Description

This PR marks Lighthouse v0.2.0. 

This release marks the stable version of Lighthouse, ready for the approaching Medalla testnet.

### CODE REVIEW

**Comment by AgeManning** [comment]
Closing in favour of #1452

**Comment by AgeManning** [comment]
Reopening this to version bump crates. 

Deceptive when compiling

File: beacon_node/version/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
```
Comment (michaelsproul) [inline beacon_node/version/Cargo.toml:1]: This is a rouge file leftover from a merge

File: beacon_node/client/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "client"
-version = "0.1.2"
-authors = ["Sigma Prime <contact@sigmaprime.io>"]
+version = "0.2.0"
+authors = ["Age Manning <Age@AgeManning.com>"]
```
Comment (michaelsproul) [inline beacon_node/client/Cargo.toml:4]: Which one do you want here?

File: beacon_node/beacon_chain/Cargo.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "beacon_chain"
-version = "0.1.2"
-authors = ["Paul Hauner <paul@paulhauner.com>"]
+version = "0.2.0"
+authors = ["Paul Hauner <paul@paulhauner.com>", "Age Manning <Age@AgeManning.com>"]
```
Comment (michaelsproul) [inline beacon_node/beacon_chain/Cargo.toml:4]: I think Paul removed these, not sure if you want to keep them

File: beacon_node/Cargo.toml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "beacon_node"
-version = "0.1.2"
-authors = ["Paul Hauner <paul@paulhauner.com>", "Sigma Prime <contact@sigmaprime.io>"]
+version = "0.2.0"
+authors = ["Paul Hauner <paul@paulhauner.com>", "Age Manning <Age@AgeManning.com"]
```
Comment (michaelsproul) [inline beacon_node/Cargo.toml:4]: Another

**Pull request closed without merging**